docs/topics/Gateway.md

```diff
@@ -426,6 +426,17 @@ Sent when a channel is updated. The inner payload is a [guild channel](#DOCS_CHA
 
 Sent when a channel relevant to the current user is deleted. The inner payload is a [DM](#DOCS_CHANNEL/channel-object) or [Guild](#DOCS_CHANNEL/channel-object) channel object.
 
+### Channel Pins Update
+
+Sent when a message is pinned or unpinned in a text channel. This is not sent when a pinned message is deleted.
+
+###### Channel Pins Update Event Fields
+
+| Field | Type | Description |
+|-------|------|-------------|
+| channel_id | snowflake | the id of the channel |
+| last_pin_timestamp? | ISO8601 timestamp | the time at which the most recent pinned message was pinned |
```

Comment:
I did notice some places where it's in the type. I took a guess at which was correct.